config.yaml

```diff
@@ -433,3 +445,11 @@ repositories:
       hiero-sdk-go-maintainers: maintain
       hiero-sdk-go-committers: write
     visibility: public
+  - name: hiero-docs
```

Comment:
@theekrystallee please remove the repository section. This would create a new and empty repo. We need to add that section after the repo transfer. @rbarkerSL and his team will take care of that.